osu.Game/Screens/Edit/Editor.cs

```diff
@@ -275,11 +275,22 @@ protected override bool OnKeyDown(KeyDownEvent e)
 
         protected override bool OnScroll(ScrollEvent e)
         {
-            scrollAccumulation += (e.ScrollDelta.X + e.ScrollDelta.Y) * (e.IsPrecise ? 0.1 : 1);
+            const double precision = 1;
 
-            const int precision = 1;
+            double scrollComponent = e.ScrollDelta.X + e.ScrollDelta.Y;
 
-            while (Math.Abs(scrollAccumulation) > precision)
+            double scrollDirection = Math.Sign(scrollComponent);
+
+            // this is a special case to handle the "pivot" scenario.
+            // if we are precise scrolling in one direction then change our mind and scroll backwards,
+            // the existing accumulation should be applied in the inverse direction to maintain responsiveness.
+            if (Math.Sign(scrollAccumulation) != scrollDirection)
+                scrollAccumulation = scrollDirection * (precision - Math.Abs(scrollAccumulation));
```

Comment:
Did you want to abs this again? like `scrollDirection * Math.Abs(precision - Math.Abs())`?

Just thinking if you have a large negative accumulation and change to positive, the calc is going to be `1 * (1 - (>1)) = 1 * (<0) = <0`